README.md

```diff
@@ -111,24 +111,13 @@ You supply either an IAM service **API key** or an **access token**:
 ```go
 // In the constructor, letting the SDK manage the IAM token
 import "github.com/watson-developer-cloud/go-sdk/discoveryv1"
-
+authenticator := &core.IamAuthenticator{
+	ApiKey: "<apikey>",
+}
 discovery, discoveryErr := discoveryv1.NewDiscoveryV1(&discoveryv1.DiscoveryV1Options{
 		URL:       "<service_url>",
 		Version:   "2018-03-05",
-		IAMApiKey: "<apikey>",
-	})
-```
-
-**Supplying the access token**
-
-```go
-// In the constructor, assuming control of managing IAM token
-import "github.com/watson-developer-cloud/go-sdk/discoveryv1"
-
-discovery, discoveryErr := discoveryv1.NewDiscoveryV1(&discoveryv1.DiscoveryV1Options{
-		URL:            "<service_url>",
-		Version:        "2018-03-05",
-		IAMAccessToken: "<IAM_access_token>",
+		Authenticator: authenticator,
```

Comment:
extra comma? or are these ok in go? im guessing they're ok because I see them around

Reply:
no, its ok

Reply:
in fact, it's required :)